src/popsift/sift_conf.cu

```diff
@@ -8,12 +8,25 @@
 #include "common/debug_macros.h"
 #include "sift_conf.h"
 
-#include <boost/algorithm/string.hpp>
-
 #include <iostream>
+#include <algorithm>
 
 using namespace std;
 
+static bool stringIsame( string l, string r )
+{
+    std::for_each( l.begin(), l.end(), [](char& c) { c = ::tolower(c); });
```

Comment:
```suggestion
    // early exit
    if(l.size() != r.size())
    {
        return false;
    }

    std::for_each( l.begin(), l.end(), [](char& c) { c = ::tolower(c); });
```

Reply:
I take the proposed change. I wanted to go for strncmp, but Windows does apparently not have it, it has stricmp. And adding Boost just for this seemed like too much.

Reply:
@simogasp It would be really preferable to use strcasecmp() for non-Windows and _stricmp() for Windows. The first requires inclusion of strings.h (at least on Linux and Mac) and the other string.h. The first one is a function, the second is a macro. I'm too stupid to write a CMake test, so I gave up and wrote these crappy lines to make it run.
And is speed essential? It should be used to evaluate the command line.

Reply:
I think it is ok like this, it is just for comparing the string in the enum. And I agree that it's not worth to re-add boost just for this.